### PR TITLE
Initial pot/potluck report

### PR DESCRIPTION
**Pull request by hny-gd**

### CODE REVIEW

File: website/content/en/status/report-2023-01-2023-03/pot.adoc
Comment (pauamma) [inline website/content/en/status/report-2023-01-2023-03/pot.adoc:16]: s/been added]/been added],/ (serial comma)

File: website/content/en/status/report-2023-01-2023-03/pot.adoc
Comment (pauamma) [inline website/content/en/status/report-2023-01-2023-03/pot.adoc:18]: "how easy it should be"? Or "how easy it is"? To me, the former hints that it might not be as easy as it ought to.

**Comment by hny-gd** [comment]
Thanks a lot for your feedback,  @pauamma, your suggested changes have been committed.

**Review by lsalvadore — APPROVED** [review]
Thanks @hny-gd for your report, and thanks to @pauamma too for his review. The report looks good to me.

I add my mentors @sergio-carlavilla and @dbaio to the reviewers to get the necessary mentor approval before merging,

**Review by pauamma — APPROVED** [review]

File: website/content/en/status/report-2023-01-2023-03/pot.adoc
Comment (dbaio) [inline website/content/en/status/report-2023-01-2023-03/pot.adoc:4]: To note:

`link:https://github.com/bsdpot[https://github.com/bsdpot]`
This is redundant, don't need to repeat.
`link:https://github.com/bsdpot[]`


Could you please fix this in the template when you have time? It's coming from there.

I won't bother you with this in the following reports.

and you can proceed here.

Thanks.

**Review by dbaio — APPROVED** [review]

File: website/content/en/status/report-2023-01-2023-03/pot.adoc
Comment (lsalvadore) [reply website/content/en/status/report-2023-01-2023-03/pot.adoc:4]: Sure, I will fix the template.

I will also remove the redundancy from all reports (at least of this quarter), but I will probably do it in one single commit for all of them at once rather than fixing them one by one as they come.

**Comment by lsalvadore** [comment]
@hny-gd: Your commit does not uses a real email address. Indeed it has

`Stephan Lichtenauer <65355462+hny-gd@users.noreply.github.com>`

Would like to share a real email address? I recommend doing so, especially if you contribute or plan to contribute regularly to FreeBSD. I would also add you to https://docs.freebsd.org/en/articles/contributors/ using that same email address (I can also add you without any email address, if you prefer).

**Comment by hny-gd** [comment]
> @hny-gd: Your commit does not use a real email address. Indeed it has
> 
> `Stephan Lichtenauer <65355462+hny-gd@users.noreply.github.com>`
> 
> Would like to share a real email address? I recommend doing so, especially if you contribute or plan to contribute regularly to FreeBSD. I would also add you to https://docs.freebsd.org/en/articles/contributors/ using that same email address (I can also add you without any email address, if you prefer).

Of course, my email address is sl@honeyguide.eu (it already had been listed in the past reports for the Potluck part, but since Bretton is doing all the work meanwhile, I wanted to make this clear this time).

Feel free to list my name together with the mail address on the contributors page, thank you very much for that, @lsalvadore!

**Comment by lsalvadore** [comment]
Merged with commit https://github.com/freebsd/freebsd-doc/commit/eb2e55d843695eb0c5b4d3f289a72f6b07543581, thanks.

@hny-gd: I have also opened review https://reviews.freebsd.org/D39361 to add you to the list of additional FreeBSD contributors.

**Comment by hny-gd** [comment]
Thanks, @lsalvadore!

**Pull request closed without merging**
